### ISSUE TITLE
任务的创建地方在哪，应用部署一般是集群，是不是需要指定或者负载方式访问应用

### ISSUE THREAD

**Issue by zwp201301** [rabbitgyk/uncode-schedule#1]
我目前的理解是，任务创建的网页就只有指定的类和方法，无法跨应用。请教能不能http方式，通过任务管理器去访问任务执行的应用。

**Comment by rabbitgyk** [comment]
任务是写在应用中的java方法，任务的管理主要通过zookeeper管理的

**Comment by zwp201301** [comment]
有空方便整个示例程序不。service方法注册到zk里，然后通过页面来管理任务。是不是部署两个应用。一个是具体的业务应用；另外一个是任务管理应用。

**Comment by rabbitgyk** [comment]
是的  管理应用的系统是单独部署的，https://github.com/rabbitgyk/uncode-schedule-learn 这是一个示例工程
